test/integration/gateways/mercadopago_test.exs

```diff
@@ -2,35 +2,136 @@ defmodule Gringotts.Integration.Gateways.MercadopagoTest do
   # Integration tests for the Mercadopago 
 
   use ExUnit.Case, async: false
-  alias Gringotts.Gateways.Mercadopago
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+  alias Gringotts.Gateways.Mercadopago, as: Gateway
 
   @moduletag :integration
 
-  setup_all do
-    Application.put_env(:gringotts, Gringotts.Gateways.Mercadopago,
-      [ 
-        public_key: "your_secret_public_key",
-        access_token: "your_secret_access_token"
-      ]
-    )
-  end
-  
-  # Group the test cases by public api
-  describe "purchase" do
-  end
+  @amount Money.new(45, :BRL)
+  @sub_amount Money.new(30, :BRL)
+  @good_card %Gringotts.CreditCard{
+            first_name: "Hermoine",
+            last_name: "Granger",
+            number: "4509953566233704",
+            year: 2030,
+            month: 07,
+            verification_code: "123",
+            brand: "VISA"
+        }
 
-  describe "authorize" do
-  end
+  @bad_card %Gringotts.CreditCard{
+            first_name: "Hermoine",
+            last_name: "Granger",
+            number: "4509953566233704",
+            year: 2000,
+            month: 07,
+            verification_code: "123",
+            brand: "VISA"
+        }
 
-  describe "capture" do 
-  end
+  @good_opts [email: "hermoine@granger.com",
+         order_id: 123126,
+         customer_id: "311211654-YrXF6J0QikpIWX",
+         config: [access_token: "TEST-2774702803649645-031303-1b9d3d63acb57cdad3458d386eee62bd-307592510",
+                 public_key: "TEST-911f45a1-0560-4c16-915e-a8833830b29a"],
+         installments: 1
+        ]
+  @new_cutomer_good_opts [order_id: 123126,
+         config: [access_token: "TEST-2774702803649645-031303-1b9d3d63acb57cdad3458d386eee62bd-307592510",
+                 public_key: "TEST-911f45a1-0560-4c16-915e-a8833830b29a"],
+         installments: 1
+        ]
+  @new_cutomer_bad_opts [order_id: 123126,
```

Comment:
The only difference between `@new_cutomer_good_opts` and `@new_cutomer_bad_opts` is in the `:config` key. Please do not test the presence or absence of **required_fields** in your `opts` because the tests for that have been written in `gringotts_test.exs`